mipengine/node/tasks/udfs.py

```diff
@@ -498,6 +498,7 @@ def _generate_udf_statements(
         {
             "udf_name": udf_name,
             "node_id": node_config.identifier,
+            "min_row_count": node_config.privacy.minimum_row_count,
```

Comment:
`min_row_count` should not be hardcoded here.

Reply:
What do you mean hard coded? It's read from the config file.

Reply:
Oh, you mean the string. I see.